package.json

```diff
@@ -4,28 +4,32 @@
   "private": true,
   "dependencies": {
     "@openstax/open-search-client": "./open-search-client/",
+    "@types/pretty": "^2.0.0",
     "ally.js": "^1.4.1",
     "color": "^3.1.0",
     "details-element-polyfill": "^2.3.1",
     "history": "^4.7.2",
     "html-entities": "^1.2.1",
+    "jest": "24.7.1",
     "lodash": "^4.17.11",
-    "path-to-regexp": "^2.4.0",
+    "path-to-regexp": "^3.0.0",
+    "pretty": "^2.0.0",
```

Comment:
:shipit:

Reply:
moved to dev-dependencies because its only used for test output